### DISCUSSION TITLE
Pwpush Kubernetes Deployment

### DISCUSSION THREAD

**Discussion by weyderfs** [pglombardo/PasswordPusher#2364] [Show and tell]
Hi guys!

I've been created a Kubernetes deployment using Kustomize for Pwpush. It's like a MVP `v0`. In the future I intend to create a Helm Chart to improve the experience of deployment.

Everyone is welcome to contribute with the project https://github.com/weyderfs/pwpush-k8s

**Comment by pglombardo** [comment]
Hi @weyderfs - this a great contribution!  Much appreciated.  I filed #2370 to add a link to the installation document soon.

**Reply by weyderfs** [reply]
Hi @pglombardo Thank you for providing this incredible solution for sharing sensitive data.

Regarding the issue you opened, will you need help adding documentation? You can use my [README.md](https://github.com/weyderfs/pwpush-k8s/blob/main/README.md)

**Reply by fiskhest** [reply]
This is a step in the right direction! Imo, the linked suggestions makes quite some heavy assumptions right now, including not supporting persistent deployments.

**Comment by pglombardo** [comment]
Added the project to the [3rd party tools documentation](https://docs.pwpush.com/docs/3rd-party-tools/#deployment).

**Comment by pglombardo** [comment]
Hi @weyderfs - new K8s, Kustomize & Helm documentation is live with links to your repository.  Thanks for the contribution and help!

https://docs.pwpush.com/docs/self-hosted-kubernetes/
